openapi.yaml

```
@@ -3478,20 +3481,28 @@ paths:
       - url: https://api.linode.com/v4beta
       tags:
       - Images
-      summary: Machine Image Upload
+      summary: Image Upload
       description: |
-        Initiates a Machine Image upload.
+        Initiates an Image upload.
+
+        This endpoint creates a new private Image object and returns it along
+        with the URL to which image data can be uploaded.
 
-        This endpoint creates a new private Image object and returns it, along
-        with the URL the image data should be uploaded to.
+        - Image data must be uploaded within 24 hours of creation or the
+        upload will be cancelled and the image deleted.
 
-        - Machine Images must be uploaded within 24 hours of creation or the upload will be cancelled.
+        - Image uploads should be made as an HTTP PUT request to the returned `upload_url`, with a
+        `Content-type: application/octet-stream` header included in the request. For example:
 
-        - Machine Image uploads should be made as an HTTP PUT request to the returned URL, with a
-        `Content-type: application/octet-stream` header included in the request.
+              curl -v \
+                -H "Content-Type: application/octet-stream" \
+                --upload-file example.img.gz \
+                "upload_url" \
```

I would reference this as a shell variable:

```suggestion
                $UPLOAD_URL \
```